recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```

Comment:
I think during one of the last meeting we agreed to explicitly depend on GCC and LLVM (OSX), isn't it? If we don't we still get https://github.com/conda-forge/matplotlib-feedstock/issues/61 on systems which does not have libgcc or gcc installed.